modules/apps/27-interchain-accounts/controller/types/msgs.go

```diff
@@ -46,3 +48,24 @@ func (msg MsgRegisterAccount) GetSigners() []sdk.AccAddress {
 
 	return []sdk.AccAddress{accAddr}
 }
+
+// NewMsgSubmitTx creates a new instance of MsgSubmitTx
+func NewMsgSubmitTx(connectionID, owner string, timeoutHeight clienttypes.Height, timeoutTimestamp uint64, packetData *icatypes.InterchainAccountPacketData) *MsgSubmitTx {
```

Comment:
I find it mildly odd that the argument order doesn't match the protobuf numbering order. I don't have a strong preference. Typically timeout info is last though